front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -61,7 +64,11 @@ export function AgentSuggestion({
                   key={`message-${userMessage.sId}-suggestion-${agent.sId}`}
                   label={agent.name}
                   visual={agent.pictureUrl}
-                  onClick={() => selectSuggestionHandler(agent)}
+                  onClick={() => {
```

Comment:
would put the setLoading here (separate loading logic from selecting logic) and add a com "Loading state to avoid erroneous double click triggering multiple retries" => since it's not obvious why we do that

Reply:
:+1: updated